### DISCUSSION TITLE
How to format Date and Time column

### DISCUSSION THREAD

**Discussion by egwrk** [microsoft-search/pnp-modern-search#3773] [Q&A] [answered]
I'm new to HTML handlebars and I'm trying to use it to format the dates in my PnP - Search Results web part but with no luck. By default, the date is displayed like this: 2024-06-19T04:00:00Z

Does anyone know how to format the date so it displays like this instead: 19 June 2024
I'm using the details list layout and also prefer not to use an HTML template.

I've tried using each of the lines below but the date column goes blank:

```
{{getDate HolidayDate "dd MMM yyyy"}}

{{getDate slot item @root.slots.HolidayDate "dd MMM yyyy"}}

{{getDate HolidayDate dateFormat "dd MMM yyyy"}}

{{getDate slot item @root.slots.HolidayDate dateFormat "dd MMM yyyy"}}
```

**Comment by egwrk** [comment, marked as answer]
Found the answer here: https://github.com/microsoft-search/pnp-modern-search/discussions/3468

This is the syntax I used and it worked! 
{{getDate item.RefinableDate17 'DD MMMM yyyy' 0 true}}

**Comment by kasperbolarsen** [comment]
I use this overview a lot, https://docs.celigo.com/hc/en-us/articles/360045564992-Handlebar-expressions-for-date-and-time-format-codes
